modules/types/src/index.ts

```diff
@@ -2,5 +2,19 @@
 // SPDX-License-Identifier: MIT
 // Copyright (c) vis.gl contributors
 
-export type {TypedArray, TypedArrayConstructor, NumericArray, NumberArray} from './array-types';
+export type {
+  TypedArray,
+  TypedArrayConstructor,
+  NumericArray,
+  NumberArray,
```

Comment:
Could we remove `NumberArray`? I don't see why we need an alias for `NumericArray` and it feels off to have `NumberArray` support typed arrays while `NumberArrayX` explicitly does not

Reply:
> Could we remove NumberArray? 

Unfortunately it is used a lot, so that would best be done in a major release. 

> I don't see why we need an alias for NumericArray

No, we don't need two types. I was actually trying to phase out `NumericArray`, but take your point on other `NumberArrayX` not supporting typed arrays, so perhaps I should start phasing out `NumberArray` instead.